pinot-tools/src/main/java/org/apache/pinot/tools/data/generator/DataGenerator.java

```diff
@@ -163,26 +174,75 @@ private FieldSpec buildSpec(DataGeneratorSpec genSpec, String column) {
 
   public static void main(String[] args)
       throws IOException {
-    final String[] columns = {"column1", "column2", "column3", "column4", "column5"};
+
     final Map<String, DataType> dataTypes = new HashMap<>();
     final Map<String, FieldType> fieldTypes = new HashMap<>();
     final Map<String, TimeUnit> timeUnits = new HashMap<>();
 
     final Map<String, Integer> cardinality = new HashMap<>();
     final Map<String, IntRange> range = new HashMap<>();
     final Map<String, Map<String, Object>> template = new HashMap<>();
-
-    for (final String col : columns) {
-      dataTypes.put(col, DataType.INT);
-      fieldTypes.put(col, FieldType.DIMENSION);
-      cardinality.put(col, 1000);
+    Map<String, Double> mvCountMap = new HashMap<>();
+    Map<String, Integer> lengthMap = new HashMap<>();
+    List<String> columnNames = new ArrayList<>();
+
+    int cardinalityValue = 5;
+    int strLength = 5;
+
+    String colName = "colInt";
+    dataTypes.put(colName, DataType.INT);
```

Comment:
Sorry, I don't follow these changes. All of this information (column name, type etc) should come from the data characteristics file. What is colInt then?

Reply:
Apparently this was a test for data generator. Just wanted to add more cases to it.